src/main/java/org/folio/rest/impl/DeleteOrderLineByIdHelper.java

```diff
@@ -24,15 +22,15 @@
     setDefaultHeaders(httpClient);
   }
 
-  void deleteLine(String orderId, String lineId) {
-    getPoLineByIdAndValidate(orderId, lineId)
+  void deleteLine(String lineId) {
+    HelperUtils.getPoLineById(lineId, lang, httpClient,ctx, okapiHeaders, logger)
```

Comment:
It seems for delete case no need to get line from storage. Probably better to call `deletePoLine`

Reply:
This way we will not be able to delete poline sub objects because of having `polineId` as available data only. We should make additional call.